### PR TITLE
Enable XML documentation output for supporting projects

### PR DESCRIPTION
**Pull request by Nihlus**
This PR enables XML documentation output for the supporting projects, and corrects a number of compile-time warnings raised by incorrect or missing documentation.

This removes a substantial number of compiler warnings, and improves the documentation coverage.

### CODE REVIEW

File: src/Generator.Bind/Generator.Bind.csproj [hunk 1]
```diff
@@ -63,6 +64,7 @@
     <ConfigurationOverrideFile>
     </ConfigurationOverrideFile>
     <DefineConstants>TRACE;</DefineConstants>
+    <DocumentationFile>bin\Debug\Bind.xml</DocumentationFile>
```
Comment (Frassle) [inline src/Generator.Bind/Generator.Bind.csproj]: Release

File: src/Generator.Converter/Generator.Convert.csproj [hunk 2]
```diff
@@ -63,6 +64,7 @@
     <ConfigurationOverrideFile>
     </ConfigurationOverrideFile>
     <DefineConstants>TRACE;</DefineConstants>
+    <DocumentationFile>bin\Debug\Convert.xml</DocumentationFile>
```
Comment (Frassle) [inline src/Generator.Converter/Generator.Convert.csproj]: Release

File: src/Generator.Rewrite/Generator.Rewrite.csproj [hunk 3]
```diff
@@ -34,6 +35,7 @@
     <Optimize>true</Optimize>
     <OutputPath>bin\Release\</OutputPath>
     <DefineConstants>TRACE</DefineConstants>
+    <DocumentationFile>bin\Debug\Rewrite.xml</DocumentationFile>
```
Comment (Frassle) [inline src/Generator.Rewrite/Generator.Rewrite.csproj]: Release

**Comment by Nihlus** [comment]
Oops, missed those. The dangers of copy & paste.

**Pull request merged**